frontend/src/components/maps/leaflet/InfoSlideOut/InfoContent.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { ListGroup } from 'react-bootstrap';
+import { IBuilding, IParcel, PropertyTypes } from 'actions/parcelsActions';
+import { Label } from 'components/common/Label';
+import { ParcelPIDPIN } from './ParcelPIDPIN';
+import ParcelAttributes from './ParcelAttributes';
+import './InfoSlideOut.scss';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import Claims from 'constants/claims';
+
+export type IInfoContent = {
```

Comment:
The I makes a bit less sense if you aren't using an interface, but I don't think it really matters.